tests/integration/test_storage_kafka/test.py

```diff
@@ -452,10 +475,15 @@ def test_kafka_formats(kafka_cluster):
         # },
     }
```

Comment:
Ugh, this should be rewritten as @pytest.mark.parametrize, @oandrew do you have passion to do this? (this will make test for each format as a separate)

Reply:
Maybe with next PR.